### PR TITLE
Minimal `winget()` implementation

### LINKED ISSUE #192: `winget()` not implemented
http://www.byond.com/docs/ref/#/proc/winget

NTstation

### PR DESCRIPTION
**Pull request by wixoaGit**
Implements `winget()`. Currently only supports the `size` and `is-disabled` properties.

I didn't use RT's serialization manager to read the property because I want to move UI away from using it. It's not great for setting singular properties and reading properties that are constantly changing like `size` and `splitter`.

Also implemented the CHILD control's `splitter` property. This PR is enough to get the "Fit Viewport" verb working.

Fixes #192

### CODE REVIEW

**Comment by amylizzle** [comment]
Easy one to add to this would be `hwmode` since for us it should always return true. Also it's used in goon.

**Comment by wixoaGit** [comment]
> Easy one to add to this would be `hwmode` since for us it should always return true. Also it's used in goon.

Done

File: OpenDream.sln.DotSettings [hunk 1]
```diff
@@ -21,6 +21,8 @@
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=deciseconds/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=fcopy/@EntryIndexedValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/UserDictionary/Words/=flist/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=hwmode/@EntryIndexedValue">True</s:Boolean>
+
```
Comment (wixoaGit) [inline OpenDream.sln.DotSettings:25]:
```suggestion
	<s:Boolean x:Key="/Default/UserDictionary/Words/=hwmode/@EntryIndexedValue">True</s:Boolean>
```

File: OpenDreamClient/Interface/InterfaceElement.cs [hunk 2]
```diff
@@ -26,6 +26,17 @@ public class InterfaceElement {
         UpdateElementDescriptor();
     }
 
+    /// <summary>
+    /// Attempt to get a DMF property
+    /// </summary>
+    /// <param name="property"></param>
+    /// <param name="value"></param>
+    /// <returns></returns>
```
Comment (wixoaGit) [inline OpenDreamClient/Interface/InterfaceElement.cs:34]:
```suggestion
```

**Pull request merged**